app/services/s3_zip_downloader.rb

```diff
@@ -0,0 +1,33 @@
+class S3ZipDownloader
+  require 'zip'
+
+  def initialize(claim)
+    @claim = claim
+    @folder = Dir.mktmpdir("#{@claim.case_number}-", './tmp')
```

Comment:
cool, perhaps a
```
Rails.root.join('tmp') => Pathname object
```
or
```
File.join(Rails.root, 'tmp') => string
```

Reply:
but just a suggestion

Reply:
I just used the standard ruby invocation of https://apidock.com/ruby/Dir/mktmpdir/class